app/lib/package/models.dart

```diff
@@ -91,6 +91,9 @@ class Package extends db.ExpandoModel {
     doNotAdvertiseFlag = value;
   }
 
+  @db.BoolProperty()
+  bool isFlutterFavorite;
```

Comment:
Add a TODO to make this required after the backfill updated the entities in Datastore.